packages/components/src/popover/index.tsx

```diff
@@ -175,17 +174,23 @@ const UnforwardedPopover = (
 		placement: placementProp = 'bottom-start',
 		offset: offsetProp = 0,
 		focusOnMount = 'firstElement',
-		anchorRef,
-		anchorRect,
-		getAnchorRect,
+		anchor,
 		expandOnMobile,
 		onFocusOutside,
 		__unstableSlotName = SLOT_NAME,
 		flip = true,
 		resize = true,
 		shift = false,
-		__unstableShift,
+
+		// Deprecated props
 		__unstableForcePosition,
+		__unstableShift,
```

Comment:
How is the deprecation of `__unstableShift` going to be handled? (Like when is it going to be removed)

Reply:
We introduced a new prop called `shift`. The `__unstableShift` prop was deprecated and will be deleted in WordPress 6.3, but until that moment we're ensuring back-compat (see https://github.com/WordPress/gutenberg/pull/43845)